docs/builders/vite.md

```diff
@@ -53,15 +53,31 @@ When Storybook loads, it automatically merges the configuration into its own. Ho
 
 <!-- prettier-ignore-end -->
 
-The asynchronous function `viteFinal` receives a `config` object with the default builder configuration and returns the updated configuration.
+The asynchronous function [`viteFinal`](../api/main-config-vite-final.md) receives a `config` object with the default builder configuration and returns the updated configuration.
 
-You can also override the builder's configuration based on the environment. For instance, if you need to provide a custom configuration for development purposes and another for production, you can extend the default configuration as follows:
+### Environment-based configuration
+
+If you need to customize the builder's configuration and apply specific options based on your environment, extend the `viteFinal` function as follows:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/main-config-vite-final-env.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### Override the default configuration
+
+By default, the Vite builder in Storybook searches for the Vite configuration file in the root directory of your Storybook project. However, you can customize it to look for the configuration file in a different location. For example:
```

Comment:
you could add an extra note with something like this:

> this is also handy if you do _not_ want Storybook to automatically load your Vite configuration file, as you can use this to point it to a file that doesn't exist.

This is a use case we hear often in Discord.